test/Project.toml

```diff
@@ -8,6 +8,7 @@ ClimaAnalysis = "29b5916a-a76c-4e73-9657-3c8fd22e65e6"
 ClimaAtmos = "b2c96348-7fb7-4fe0-8da9-78d88439e717"
 ClimaComms = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
 ClimaCore = "d414da3d-4745-48bb-8d80-42e94e092884"
+ClimaCoreMakie = "908f55d8-4145-4867-9c14-5dad1a479e4d"
 ClimaCorePlots = "cf7c7e5a-b407-4c48-9047-11a94a308626"
```

Comment:
Can we remove ClimaCorePlots and Plots from the test environment now too?

Reply:
I see that we have some commented-out plotting lines in regridder_tests.jl. It's fine to remove those lines since they're not being used and we want to switch to ClimaUtilities regridding soon anyway. That might be the only place we use Plots/ClimaCorePlots in test/